apps/pie-docs/src/assets/styles/components/_contentPage.scss

```diff
@@ -7,17 +7,17 @@
     min-height: 100%;
 
     > * {
-        padding-left: f.spacing(d);
-        padding-right: f.spacing(d);
+        padding-inline-start: var(--dt-spacing-d);
```

Comment:
nit but could these be shortened to `padding-inline: var(--dt-spacing-d) var(--dt-spacing-d)` or maybe just `padding-inline: var(--dt-spacing-d)`? I think If one value is given, it is used as the value for both inline start and end 😄

Reply:
It's a great shout @LTurns but I think we have to avoid this specific shorthand because it's not supported by Safari 14 😞

Reply:
good point! i will address this in the next PR.

Reply:
@xander-marjoram I see we already use them several times in the codebase, so we probably need to clean that up at some point

Reply:
oh I see, thank you! I've seen some `padding-inline` and `padding-block` shortcodes dotted around (just spotted one [here](https://github.com/justeattakeaway/pie/pull/1180/files#diff-a1a7b39ce88d57776b4d4176d9777d9a27fad605af9debe8c30e911e41290b1fL37)) - can defo be removed in a different PR though 🎉

Reply:
oh sorry @raoufswe just seen you beat me to it! they do seem to be in lots of different areas - shall we make a ticket for this? I'm happy to write something up 👍

Reply:
@LTurns yea that would be great! thanks